release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2441,6 +2450,56 @@ providerSpec:
 
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1868751[*BZ#1868751*].
 
+[[ocp-4-5-8-added-credentialsmode-parameter-for-aws-install-workaround]]
```

Comment:
I updated the headers in my PR to reflect what we talked about last week. Do you want to update yours to match?: `[id="ocp-4-5-8-added-credentialsmode-parameter-for-aws-install-workaround"]`

Reply:
oh oops yup! will do :)